js/index.js

```diff
@@ -46,3 +46,9 @@ const carrouselImage = document.querySelector(".carrousel__img");
 carrouselArrows.forEach(function(element) {
   element.addEventListener("click", carrousel);
 });
+/* Function about the arrow down or drop down wherever you want
+*/
+function goDown(){
+  var lookdown = document.getElementById('carrousel-section');//take the destini to scroll
```

Comment:
please remember the structure of comments.

Reply:
I think you shouldn't use **var** as a good practice, you can use **let.**
I also think we agreed that we would use **arrow functions**.
Please, if I'm wrong, correct me.